tests/integration/cmd_push_test.go

```diff
@@ -85,8 +119,6 @@ var _ = Describe("odo push command tests", func() {
 			Expect(stdOut).To(ContainSubstring(("exampleDir")))
 		})
 
-		/* TODO uncomment once https://github.com/openshift/odo/issues/1354 is resolved
-
 		It("should build when a file is deleted from the directory", func() {
```

Comment:
Remove this scenario as you have implemented the same scenario in a better way, otherwise it will be a duplicate scenario of https://github.com/openshift/odo/pull/1999/files#diff-e977fd6c6dedc4d8ed7367e136764033R56.